offchain/authority-claimer/src/main.rs

```diff
@@ -3,18 +3,17 @@
 
 use authority_claimer::config::Config;
 use std::error::Error;
-use tracing_subscriber::filter::{EnvFilter, LevelFilter};
+use tracing::info;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn Error>> {
+    // Getting the configuration.
+    let config: Config = Config::new().map_err(Box::new)?;
+
     // Settin up the logging environment.
-    let env_filter = EnvFilter::builder()
-        .with_default_directive(LevelFilter::INFO.into())
-        .from_env_lossy();
-    tracing_subscriber::fmt().with_env_filter(env_filter).init();
+    logs::configure(&config.authority_claimer_config.logs_config);
 
-    // Getting the configuration.
-    let config = Config::new().map_err(Box::new)?;
+    info!(?config, "Starting Authority Claimer");
```

Comment:
```suggestion
```

This line looks like a duplicate of the following one:

https://github.com/cartesi/rollups-node/blob/e9a9c23299a79a294d2d5ad85099a8bcdae94bfe/offchain/authority-claimer/src/lib.rs#L25:

Reply:
@fmoura FYI